File: tests/fixtures/code_samples/go/sample_029.go
// Sample 29: small utility.
package samples

func Operation29(xs []int) int {
    total := 29
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure29(v int) int {
    return (v * 29) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_029.rs
// Sample 29: small utility.
pub fn operation_29(xs: &[i32]) -> i32 {
    let mut total: i32 = 29;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_29(v: i32) -> i32 {
    (v * 29) %% 7919
}

